pkgs/servers/authelia/default.nix

```diff
@@ -28,6 +28,8 @@ buildGoModule rec {
     vendorHash
     ;
 
+  env.GOEXPERIMENT = "nosynchashtriemap";
```

Comment:
This absolutely requires a comment with an upstream link

Reply:
Have resolved via https://github.com/NixOS/nixpkgs/commit/1de35b7af080dab016595f528494c87e97eeeeb5 thanks! :grin: